.eslintrc.json

```diff
@@ -205,6 +205,13 @@
       "files": ["*.ts"],
       "extends": ["plugin:@ngrx/component-store-strict"]
     },
+    {
+      "files": ["apps/**/app/**/*.html", "libs/**/*.html"],
```

Comment:
Question: what about components using inline templates?

Reply:
It looks like the parser used in the next line also checks inline templates. :)

Reply:
I tested a few things, and the `parser` was not really needed. The inline templates are still checked, though.